chessdotcom/client.py

```diff
@@ -66,6 +66,14 @@ class Client:
 
     @classmethod
     def _do_sync_get_request(cls, resource):
+        if "user-agent" not in [
```

Comment:
I am not sure I want to add this code here since it's not required by the API

Reply:
On June 20, 2023, Chess.com published a post saying that this is now required: [Breaking Change: User-Agent Contact Info Required](https://www.chess.com/announcements/view/breaking-change-user-agent-contact-info-required)

From my relatively modest testing (and what I read), they don't have a specific, hard-coded required format, but they are blocking anything that contains "python-requests" in the user-agent header, and that is in the default header when using the requests module.

This came up because I attempted to run the original code and it got a response 403 from Chess.com, one which did not actually explain the issue at all but I eventually figured out was related to the user-agent.

I think the warning would have helped me, at least, not have to search around for an explanation, but it also doesn't actually require the user to do anything here in case Chess.com changes the API again.

Reply:
Got it! was not aware of the breaking change. Thanks for taking the time to fix it then